### PR TITLE
adding missing info to bring into compliance

### PR DESCRIPTION
**Pull request by craigcaseyMSFT**
For compliance, adding info that was missing for .NET framework 1.1

### CODE REVIEW

**Comment by craigcaseyMSFT** [comment]
Hi @gewarren  - Added some more info to these mobile capabilities files as part of bringing into compliance. Please review. Note: everything was copied over from HttpCapabilitiesBase, like we did for the summaries. 
Thanks for your help! Craig

File: xml/System.Web.Mobile/MobileCapabilities.xml [hunk 1]
```diff
@@ -342,7 +352,7 @@ public void Page_Load(Object sender, EventArgs e)
       </ReturnValue>
       <Docs>
         <summary>Gets the major version number of the wireless gateway used to access the server, if known.</summary>
-        <value>To be added.</value>
+        <value>The major version number of the wireless gateway used to access the server, if known. The default is <see langword="0" />.</value>
```
Comment (gewarren) [inline xml/System.Web.Mobile/MobileCapabilities.xml:355]:
```suggestion
        <value>The major version number of the wireless gateway used to access the server, if known. The default is 0.</value>
```

And elsewhere in this file...

File: xml/System.Web.Mobile/MobileCapabilities.xml [hunk 2]
```diff
@@ -573,7 +587,7 @@ public void Page_Load(Object sender, EventArgs e)
       </ReturnValue>
       <Docs>
         <summary>Returns the maximum length of the text that a soft-key label can display.</summary>
-        <value>To be added.</value>
+        <value>The maximum length of the text that a soft-key label can display. The default is <see langword="5" />.</value>
```
Comment (gewarren) [inline xml/System.Web.Mobile/MobileCapabilities.xml:590]:
```suggestion
        <value>The maximum length of the text that a soft-key label can display. The default is 5.</value>
```

File: xml/System.Web.Mobile/MobileCapabilities.xml [hunk 3]
```diff
@@ -1264,7 +1298,7 @@ public void Page_Load(Object sender, EventArgs e)
       </ReturnValue>
       <Docs>
         <summary>Returns the depth of the display, in bits per pixel.</summary>
-        <value>To be added.</value>
+        <value>The depth of the display, in bits per pixel. The default is <see langword="1" />.</value>
```
Comment (gewarren) [inline xml/System.Web.Mobile/MobileCapabilities.xml:1301]:
```suggestion
        <value>The depth of the display, in bits per pixel. The default is 1.</value>
```

Comment (gewarren) [reply xml/System.Web.Mobile/MobileCapabilities.xml:1301]: Same idea elsewhere in this file when it's a number...

**Comment by craigcaseyMSFT** [comment]
Hi, @gewarren  - Made the changes as requested. please let me know if anything else should be addressed before we can merge. Thanks much!

**Comment by gewarren** [comment]
Looks good, thanks Craig!

**Pull request merged**